tests/tasks/teardown/kit.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: kit-cluster-teardown
+spec:
+  description: |
+    Teardown an KIT cluster.
+    This Task can be used to teardown an KIT cluster in an AWS account.
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster which will be teared down.
+  steps:
+  - name: delete-cluster
+    image: amazon/aws-cli
+    script: |
+      kubectl version
+      aws eks update-kubeconfig --name testbed --region $(params.host-cluster-region)
+      kubectl config current-context 
+      #delete kit controlplane spec and dataplane spec crds
+      kubectl delete ControlPlane --name $(params.cluster-name)
+      kubectl delete DataPlane --name $(params.cluster-name)-nodes
```

Comment:
Once the control plane is deleted, dataplane gets deleted automatically by the KIT-operator. So we can skip here.